### PR TITLE
fix: invalid reading _tag property for error (trx)

### PR DESCRIPTION
**Pull request by dilaouid**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Check if `in` operator is available for the `error` object for the `_tag` property for `isDmkError`

### ❓ Context

- **[JIRA or GitHub link](https://ledgerhq.atlassian.net/jira/software/c/projects/LIVE/boards/465?selectedIssue=LIVE-20326)** 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by dilaouid** [comment]
Mobile e2e: https://github.com/LedgerHQ/ledger-live/actions/runs/19428650896
Desktop e2e: https://github.com/LedgerHQ/ledger-live/actions/runs/19421757608

File: libs/ledger-live-common/src/deviceSDK/tasks/core.ts [hunk 1]
```diff
@@ -79,7 +79,8 @@ type ErrorClass = CustomErrorClassType | TransportStatusErrorClassType;
 // To be able to retry a command, the command needs to take an object containing a transport as its argument
 type CommandTransportArgs = { transport: Transport };
 
-export const isDmkError = (error: any): error is DmkError => error && "_tag" in error;
+export const isDmkError = (error: unknown): error is DmkError =>
+  !!error && typeof error === "object" && error !== null && "_tag" in error;
```
Comment (francois-guerin-ledger) [inline libs/ledger-live-common/src/deviceSDK/tasks/core.ts:83]:
```suggestion
  !!error && typeof error === "object" && "_tag" in error;
```
No need to test it is truthy _and_ is not `null`
